completions/bash/multipass

```diff
@@ -339,6 +357,13 @@ _multipass_complete()
                 # TODO: Do use spaces after the completion of an option starting with "--".
                 compopt -o nospace
             ;;
+            "get")
+                _multipass_setting_keys
```

Comment:
Both `get` and `set` both take a single key argument (for the time being, at least). It would be nice to complete that argument only if not yet used (so that we're not suggesting something that doesn't work):

```
$ multipass get client.primary-name local.driver
Need exactly one setting key.
```

Reply:
Fixed.